site/content/docs/contributing/1.0-roadmap.md

```diff
@@ -15,25 +15,41 @@ as a project.
 
 High level goals for kind include:
 
-- [ ] Reach [1.0] status / 
-stability by the end of Q1 2019 / the Kubernetes 1.14 release cycle
+## Beta Requirements
+
+To reach "beta" [grade][deprecation-policy] kind needs to at minimum:
+
+- [x] Improve documentation (Though this will eternally be "In Progress" !)
+  - [X] create a documentation site - [#268]
+  - [ ] expand examples of using kind
```

Comment:
I think that @mauilion , and our community was very active in this regard, i.e.
https://github.com/charlesakalugwu/kind-examples
https://github.com/kind-ci/examples

Reply:
yes, but we need to cover some of them in our docs, especially as searching the internet for this can be hard :-)

Reply:
well, that´s a good challenge for any SEO specialist, with a project named `kind` :)

Reply:
implemented https://kind.sigs.k8s.io/docs/user/resources/ to link out to external resources, but we will want to organize more first-party resources as well